packages/patternfly-4/content/design-guidelines/usage-and-behavior/modal/design.md

```diff
@@ -18,9 +18,9 @@ Use modal dialogs load and display important information without navigating away
 ### When to use
 Use a modal dialog when information is critical and requires user input or attention.
 Use modals to:
-* [Validate user decisions](#confirmation-dialog)
+* [Validate user decisions](#get-confirmation)
```

Comment:
@jenny-s51 are these link changes also meant to be in this PR?

Reply:
no, but these changes fixed an issue on another branch that I pushed... should I change it back?